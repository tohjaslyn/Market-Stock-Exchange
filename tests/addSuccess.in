add trader1 100
exit
