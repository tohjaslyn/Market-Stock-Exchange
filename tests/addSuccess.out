$ Success.
$ Have a nice day.
